src/utils/aspect-ratio-support.tsx

```diff
@@ -26,3 +27,92 @@ export const AspectRatioSupportProvider: React.FC<Props> = ({children}) => {
 };
 
 export const useSupportsAspectRatio = (): boolean => React.useContext(AspectRatioSupport);
+
+const useAspectRatioStyles = createUseStyles(() => ({
```

Comment:
if the intension is only exporting this very vsriable, I'd call the file the same name and I'd export it as default, unless there's some reason to follow this as a standard (in that case let me know!)

Reply:
well this file has two exports know, the intention is to gather the aspect ratio functionality ins the same file